subsys/net/ip/tcp.h

```diff
@@ -110,6 +110,9 @@ struct net_tcp {
 	/** Retransmit timer */
 	struct k_timer retry_timer;
```

Comment:
Should this be removed then?

Reply:
Indeed, perhaps we could use delayed work here instead. I will investigate this option.